snippets/snippets_setup.R

```diff
@@ -0,0 +1,32 @@
+library(viridis)    # A nice color scheme for plots.
+library(ggthemes)   # Common themes to change the look and feel of plots.
+library(scales)     # Graphical scales map data to aesthetics in plots.
+library(skimr)      # Better summaries of data.
+library(lubridate)  # Date library from the tidyverse.
+library(bigrquery)  # BigQuery R client.
+library(tidyverse)  # Data wrangling packages.
+
+## CHANGE THESE AS NEEDED - default parameter values for snippets.
+DATASET <- 'aou-res-curation-output-prod.R2019Q1R2'
```

Comment:
@deflaux @calbach theoretically, even at launch users will have the ability to access multiple CDRs for analysis. At a minimum they will have access to a synthetic CDR and a real CDR. They may also have access to older CDRs for the purposes of replication. How can we design snippet setup such that it defaults to the CDR version selected at workspace setup?

Reply:
Its a great question and I don't have an answer for it right now that I like. Can you please file an issue on this repo for that and lets keep thinking about it?

For now its not too hard for people to edit it manually, and for us to update the checked-in snippet when there is a new release.

Reply:
Agreed - we discussed this last week as well. We don't have an easy fix for it so it probably wouldn't be part of the v1.

Reply:
Will do.